### PR TITLE
fix: e2e tests not deploying to ganache node

### PR DESCRIPTION
**Pull request by excaliborr**
# 🤖 Linear

Closes SAF-XXX

### CODE REVIEW

File: solidity/libraries/MerklePatriciaProofVerifier.sol [hunk 1]
```diff
@@ -51,12 +51,12 @@ library MerklePatriciaProofVerifier {
 
       // The root node is hashed with Keccak-256 ...
       if (i == 0 && rootHash != stack[i].rlpBytesKeccak256()) {
-        revert();
+        revert('revert');
```
Comment (0xOneTony) [inline solidity/libraries/MerklePatriciaProofVerifier.sol:54]: let's use some better message for the reverts

**Review by 0xOneTony — COMMENTED** [review]
looks good! 🔥 🔥 Let's add a few more one-line comments in the `Common.sol` aswell

File: solidity/libraries/MerklePatriciaProofVerifier.sol (hunk 1 above)
Comment (excaliborr) [reply solidity/libraries/MerklePatriciaProofVerifier.sol:54]: These are from the lido library and they have them as unnamed i added these when debugging, hut i think we should go back to how they have it

File: package.json [hunk 2]
```diff
@@ -10,23 +10,23 @@
...
   "scripts": {
-    "anvil:mainnet": "anvil --port 8545 -f $MAINNET_RPC --fork-block-number 18621047 --chain-id 1",
-    "anvil:optimism": "anvil --port 9545 -f $OPTIMISM_RPC --fork-block-number 112491451 --chain-id 10",
     "build": "forge build",
     "build:optimized": "FOUNDRY_PROFILE=optimized forge build",
     "coverage": "forge coverage --match-contract Unit",
     "deploy:goerli": "bash -c 'source .env && forge script -vv --rpc-url $GOERLI_RPC --slow --broadcast --private-key $DEPLOYER_GOERLI_PRIVATE_KEY solidity/scripts/DeployGoerli.s.sol:DeployGoerli'",
-    "deploy:mainnet": "bash -c 'source .env && forge script -vv --rpc-url $MAINNET_RPC --slow --broadcast --private-key $DEPLOYER_MAINNNET_PRIVATE_KEY solidity/scripts/DeployMainnet.s.sol:DeployMainnet'",
-    "deploy:optimism": "bash -c 'source .env && forge script -vv --rpc-url $OPTIMISM_RPC --slow --broadcast --private-key $DEPLOYER_OPTIMISM_PRIVATE_KEY solidity/scripts/DeployOptimism.s.sol:DeployOptimism'",
+    "deploy:mainnet": "bash -c 'source .env && forge script -vv --rpc-url $MAINNET_E2E_RPC --slow --broadcast --private-key $DEPLOYER_MAINNNET_PRIVATE_KEY solidity/scripts/DeployMainnet.s.sol:DeployMainnet'",
```
Comment (0xOneTony) [inline package.json:17]: Why is this changed? This should be called `$MAINNET_RPC` same for optimism

**Review by 0xOneTony — APPROVED** [review]
